layouts/css/layout/_helpers.styl

```diff
@@ -0,0 +1,8 @@
+.text-center
+    text-align center
+	
+.text-left
```

Comment:
As we will also have RTL language translation, better use `start` and `end` rather than `left` and `right` for `text-align`.